src/Microsoft.ML.Data/Transforms/SlotsDroppingTransformer.cs

```diff
@@ -35,7 +35,8 @@ namespace Microsoft.ML.Transforms
     /// If all the slots are to be dropped, a vector valued column will be changed to a vector of length 1 (a scalar column will retain its type) and
     /// the value will be the default value.
     /// </summary>
-    public sealed class SlotsDroppingTransformer : OneToOneTransformerBase
+    [BestFriend]
+    internal sealed class SlotsDroppingTransformer : OneToOneTransformerBase
```

Comment:
Thanks @Ivanidzo4ka ! Your changes seem good for the most part, though I might question this one in particular: if we had a feature selection estimator that used this (which I believe is one of the applications), it may be worthwhile to expose, since the most natural question one would want to ask upon fitting that is, "ok, which slots did you drop/keep", which would require exposing this anyway. But as you like. The other hidings seem good!